docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+  during its creation
+
+
+For the call reminder example, you can define a custom action `action_forget_reminders` that cancels
+all reminders:
+
+```python
+class ForgetReminders(Action):
+    """Cancels all reminders."""
+
+    def name(self) -> Text:
+        return "action_forget_reminders"
+
+    async def run(
+        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any]
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message(f"Okay, I'll cancel all your reminders.")
+
+        # Cancel all reminders
+        return [ReminderCancelled()]
+```
+
+
+:::caution
+All reminders are cancelled whenever you shutdown your Rasa server.
+:::
+
+### Defining the Training Data
+
+#### Intents and Entities
+
+You'll need to define an intent that triggers setting the reminder and one that triggers cancelling the reminder.
+The `ReminderScheduled` event
+will automatically send the intent `EXTERNAL_reminder` to your bot after  the amount of time specified,
+so you also need to add this intent to your intents. You don't need to add any training examples,
+since the intent is reserved for the reminder.
+
+```
+intents:
+- intent: EXTERNAL_reminder
+- intent: ask_remind_call
+- intent: ask_forget_reminders
+```
+
+You should add NLU training examples for scheduling and cancelling the reminder:
+
+```yaml-rasa
+nlu:
+- intent: ask_remind_call
+  examples: |
+    - remind me to call John
+    - later I have to call Alan
+    - Please, remind me to call Vova
+    - please remind me to call Tanja
+    - I must not forget to call Juste
+- intent: ask_forget_reminders
+  examples: |
+    - Forget about the reminder
+    - do not remind me
+    - cancel the reminder
+    - cancel all reminders please
+```
+
+By adding `SpacyNLP` and `SpacyEntityExtractor` to your pipeline in your config file, you won't need to annotate any of the
+names in your training data, since Spacy has a `PERSON` dimension:
+
+```yaml-rasa
+pipeline:
+# other components
+- SpacyNLP
+- SpacyEntityExtractor
+    dimensions: ["PERSON"]
+```
+
+#### Rules
+
+You'll need to add three rules: One for scheduling a reminder, one for your assistant reacting to a reminder,
+and one for cancellng a reminder.
+
+```yaml-rasa
+rules:
+- rule: Schedule a reminder
+  steps:
+  - intent: ask_remind_call
+    entities:
+    - name
+  - action: action_schedule_reminder
+
+- rule: Trigger `action_react_to_reminder` for `EXTERNAL_reminder`
+  steps:
+  - intent: EXTERNAL_reminder
+  - action: action_react_to_reminder
+
+- rule: Cancel a reminder
+  steps:
+  - intent: ask_forget_reminders
+  - action: action_forget_reminders
+```
+
+
+### Try it Out
+
+As with the external events, to try out reminders you'll need to start either [Rasa X](https://rasa.com/docs/rasa-x/)
+or a [`CallbackChannel`](your-own-website.mdx#callbackchannel).
+You'll also need to start the action server to schedule, react to, and cancel your reminders.
```

Comment:
should we give them instructions on how to start the action server? Or is it described in the readme of the reminderbot? then maybe we should just tell them to read that?

Reply:
In the readme. I'll point to it